web/src/components/storage/BootConfigField.tsx

```diff
@@ -21,59 +21,51 @@
 
 // @ts-check
 
-import React from "react";
+import React, { ReactNode } from "react";
```

Comment:
NP: I use to not import React types but use a `React.ReactType` instead. No strong opinion, but would be nice if we reach an agreement about it.